server/app/routes.py

```diff
@@ -431,3 +431,133 @@ def delete_invitation(invitation_id):
         "message": "Invitation deleted successfully.",
         "data": None
     }), 200
+
+#Routes Kantai
+
+# Inventory Routes
+
+@app.route('/inventory', methods=['GET'])
+def get_all_inventory():
+    inventory = Inventory.query.all()
+    result = []
+    for item in inventory:
+        result.append({
+            'inventory_id': item.inventory_id,
+            'product_id': item.product_id,
+            'store_id': item.store_id,
+            'quantity_received': item.quantity_received,
+            'quantity_in_stock': item.quantity_in_stock,
+            'quantity_spoilt': item.quantity_spoilt,
+            'payment_status': item.payment_status,
+            'created_at': item.created_at
+        })
+    return jsonify(result)
```

Comment:
Keep the response format consistent so that it maintains a consistent structure.
Use this as the reference:
```python
return jsonify({
            "status": "Failed",
            "message": "Email and user_id are required.",
            "data": None
        }), 400
```